app/database_etl/test_adjustment_handler/test_adjustment.py

```diff
@@ -90,10 +91,31 @@ def get_stan_model_cache(self, model_code: str, model_name: str = 'anon_model',
         return cached_model
 
     def fit_one_pystan_model(self, model_params: Dict) -> Tuple:
+        
+        
+        
+        # Adapt delta adjustment
+        
+        if model_params['n_sp'] < 10 or model_params['n_se'] < 10 or model_params['y_sp'] < 5 or model_params['y_se'] < 5:
+            delt = 0.99
+        else:
+            delt = 0.8
+            print(delt)
+            
+            
+        # Set initial values
+        
+        init = [{'prev':model_params['y_prev_obs']/model_params['n_prev_obs'], 'spec':model_params['y_sp']/model_params['n_sp'], 'sens':model_params['y_se']/model_params['n_se']}]
```

Comment:
Spread over multiple lines. Why is this a list of dicts as opposed to being a dict?

Example over multiple lines

```
init = {
  'prev' = x,
  'blah' = ...
}
```